tests/utils.rs

```diff
@@ -239,6 +239,39 @@ pub fn transfer_batch_internal(
     assert!(res.contains(&(from, codec)));
 }
 
+pub fn transform_internal(
+    mtk: &Program,
+    from: u64,
+    token_id: u128,
+    amount: u128,
+    nfts: Vec<BurnToNFT>,
+) {
+    let mut ids = vec![];
+    for burn_info in &nfts {
+        for id in burn_info.nfts_ids.clone() {
+            ids.push(id);
+        }
+    }
+    let res = mtk.send(
+        from,
+        MyMTKAction::Transform {
+            id: token_id,
+            amount,
+            nfts,
+        },
+    );
+
+    let codec = MTKEvent::Transfer {
+        operator: from.into(),
+        from: ActorId::zero(),
+        to: ActorId::zero(),
+        ids: ids.to_vec(),
```

Comment:
It is already `Vec`.
```suggestion
        ids,
```